src/request.ts

```diff
@@ -1,27 +1,33 @@
 import Config from './configuration';
 import colorize from './util/colorize';
+import cloneDeep from './util/clonedeep';
+
+// RequestInit is the type expected by `fetch()` API. 
+export interface FetchOptions extends RequestInit {
+  jwt? : string | undefined
```

Comment:
No, it extends `RequestInit`, which is the type for the `fetch()` options param, so it allows all of those plus JWT.  We then strip out the jwt [here](https://github.com/jsonapi-suite/jsorm/pull/7/files#diff-31b74031f3c83e9e883fd930aaf87f54R86) before actually making the request.